plezanje-net/web
# Peaks

Adds peaks module.

Test with plezanje-net/api#67

To test, go to: Alpinizem->Vrhovi
then click around...
Also click your way all the way to the crag (wall) page and then also to route page and see that breadcrumbs are working

closes #157
